bukkit/src/main/java/com/viaversion/viaversion/bukkit/listeners/v1_20_5to1_21/PlayerChangeItemListener.java

```diff
@@ -23,58 +23,39 @@
 import com.viaversion.viaversion.bukkit.listeners.ViaBukkitListener;
 import com.viaversion.viaversion.protocols.v1_20_5to1_21.Protocol1_20_5To1_21;
 import com.viaversion.viaversion.protocols.v1_20_5to1_21.storage.EfficiencyAttributeStorage;
-import io.papermc.paper.event.player.PlayerInventorySlotChangeEvent;
-import org.bukkit.NamespacedKey;
 import org.bukkit.enchantments.Enchantment;
 import org.bukkit.entity.Player;
 import org.bukkit.event.EventHandler;
 import org.bukkit.event.EventPriority;
 import org.bukkit.event.player.PlayerItemHeldEvent;
 import org.bukkit.inventory.ItemStack;
-import org.bukkit.inventory.PlayerInventory;
 import org.checkerframework.checker.nullness.qual.Nullable;
 
 /**
  * For some reason, mining efficiency is not calculated by the client anymore, but by the server,
  * then sending the current value to the client every time the item changes. This roughly emulates that behavior.
  */
-public final class PlayerChangeItemListener extends ViaBukkitListener {
+public class PlayerChangeItemListener extends ViaBukkitListener {
 
-    private final Enchantment efficiency = Enchantment.getByKey(NamespacedKey.minecraft("efficiency"));
-    private final Enchantment aquaAffinity = Enchantment.getByKey(NamespacedKey.minecraft("aqua_affinity"));
-    private final Enchantment depthStrider = Enchantment.getByKey(NamespacedKey.minecraft("depth_strider"));
-    private final Enchantment soulSpeed = Enchantment.getByKey(NamespacedKey.minecraft("soul_speed"));
-    private final Enchantment swiftSneak = Enchantment.getByKey(NamespacedKey.minecraft("swift_sneak"));
+    // Use legacy function and names here to support all versions
+    private final Enchantment efficiency = Enchantment.getByName("DIG_SPEED");
```

Comment:
I want these to work with Spigot's trash bytecode editing disabled, so it needs to first try the new name and then fall back to the old